apps/codeforafrica/src/components/SinglePartner/SinglePartner.js

```diff
@@ -0,0 +1,89 @@
+import { Section } from "@commons-ui/core";
+import { Box } from "@mui/material";
+import React from "react";
+
+import AboutChildPageHeader from "@/codeforafrica/components/AboutChildPageHeader";
+import ConnectBar from "@/codeforafrica/components/ConnectBar";
+import RelatedProjects from "@/codeforafrica/components/RelatedProjects";
+import RichText from "@/codeforafrica/components/RichText";
+import SectionDivider from "@/codeforafrica/components/SectionDivider";
+
+const SinglePartner = React.forwardRef(function SinglePartner(
+  { description, connect, relatedProjects, relatedProjectsTitle, name, logo },
+  ref,
+) {
+  return (
+    <Box ref={ref} sx={{ pb: { xs: 10, md: 7 } }}>
+      <AboutChildPageHeader
+        name={name}
+        image={logo}
+        FigureProps={{
+          sx: {
+            backgroundPositionY: "center",
+            backgroundRepeat: "no-repeat",
+            backgroundSize: "contain",
+            bgcolor: "background.default",
+            borderRadius: 0,
+            filter: "drop-shadow(0px 8.7px 17.4px rgba(0, 0, 0, 0.1))",
+            height: { xs: 116 },
+            width: { xs: 247 },
+          },
+        }}
+      />
+      <Section
+        sx={{
+          px: { xs: 2.5, sm: 0 },
+          pt: { xs: 2.5, md: 7 },
+          maxWidth: {
+            sm: "648px",
+            md: "912px",
+          },
+        }}
+      >
+        <RichText
+          variant="subheading"
+          component="p"
+          sx={{
+            mb: 5,
+            typography: "subheading",
+            "& > p": {
+              mb: 5,
+            },
+          }}
+          elements={description}
+        />
+
+        <ConnectBar title="Connect" links={connect} />
+      </Section>
+      {relatedProjects.length ? (
```

Comment:
```suggestion
      {relatedProjects?.length ? (
```

What if there aren't any related projects?